interpreter/src/main/scala/pl/touk/nussknacker/engine/spel/parser/DictKeyWithLabelExpressionParser.scala

```diff
@@ -0,0 +1,64 @@
+package pl.touk.nussknacker.engine.spel.parser
+
+import cats.data.Validated.{Valid, invalidNel}
+import cats.data.{NonEmptyList, Validated}
+import io.circe.parser
+import pl.touk.nussknacker.engine.api.Context
+import pl.touk.nussknacker.engine.api.context.ValidationContext
+import pl.touk.nussknacker.engine.api.definition.{AdditionalVariable => _}
+import pl.touk.nussknacker.engine.api.expression.{Expression => CompiledExpression, _}
+import pl.touk.nussknacker.engine.api.generics.ExpressionParseError
+import pl.touk.nussknacker.engine.api.typed.typing
+import pl.touk.nussknacker.engine.api.typed.typing.{Typed, TypingResult}
+import pl.touk.nussknacker.engine.graph.expression.Expression
+import pl.touk.nussknacker.engine.graph.expression.Expression.DictKeyWithLabelExpression
+import pl.touk.nussknacker.engine.spel.SpelExpressionParseError.KeyWithLabelExpressionParsingError
+
+case class DictKeyWithLabelExpressionTypingInfo(key: String, label: Option[String], typingResult: TypingResult)
```

Comment:
I see that `typingResult` is always `Unknown`. If so, let's remove it from the parameters and move to the method.